extensions/applicationinsights-properties-js/src/Context/User.ts

```diff
@@ -61,10 +61,15 @@ export class User implements IUserContext {
     constructor(config: ITelemetryConfig, core: IAppInsightsCore) {
         let _logger = safeGetLogger(core);
         let _cookieManager: ICookieMgr = safeGetCookieMgr(core);
+        let _storageNamePrefix: () => string;
 
         dynamicProto(User, this, (_self) => {
+            _self.config = config;
+            const userCookiePostfix = (_self.config.userCookiePostfix && _self.config.userCookiePostfix()) ? _self.config.userCookiePostfix() : "";
+            _storageNamePrefix = () => User.userCookieName + userCookiePostfix;
```

Comment:
nit: rename this to userCookieNameWithPostFix?

Reply:
No -- don't increase the length of the config name as this directly affects minification.
userCookiePostfix is specific enough.

Reply:
I was referring to _storageNamePrefix :smile: nvm I am ok with previous name